package.json

```diff
@@ -56,11 +56,11 @@
     "http-status-codes": "^2.0.0",
     "install": "^0.13.0",
     "ioredis": "^4.14.0",
-    "jscommons": "^4.0.0",
+    "jscommons": "^6.0.0",
     "jsonwebtoken": "^8.5.1",
     "lodash": "^4.17.4",
     "mime-types": "2.1.35",
-    "mongodb": "^4.10.0",
+    "mongodb": "^4.7.0",
```

Comment:
?

Reply:
Hi @PrinceWaune this is to ensure the project builds. New version of `jscommons` use "mongodb": "^4.7.0". Continuing to use "^4.10.0" will cause a type mismatch.

Reply:
In my opinion, we should upgrade the `MongoDB` package inside the `jscommons` to "4.10.0".

Reply:
@crazy-grizzly @ToranSharma @lydiamross

Reply:
4.7 is currently being resolved to 4.14 in the lock file